cli/cli.go

```diff
@@ -308,6 +308,27 @@ func SetupCLI(args []string) error {
 				},
 			},
 		},
+		{
+			Name:        "snapshot",
+			Description: snapshotDescription,
+			Subcommands: []cli.Command{
+				{
+					Name:  "dump",
+					Usage: "Dumps rootfs to stdout.",
+					Action: func(ctx *cli.Context) error {
+						// Expected to return ENOTTY
+						_, err := unix.IoctlGetTermios(
+							int(os.Stdout.Fd()),
+							unix.TCGETS)
+						if err == nil {
+							return errDumpTerminal
+						}
+						return runOptions.
+							CopySnapshot(ctx, os.Stdout)
+					},
+				},
+			},
+		},
```

Comment:
I think this commit is not supposed to be here. This causes the flag to be introduced twice, because this commit already exists in the history.